k4Reco/GaudiLumiCalClusterer/include/LumiCalClusterer.h

```diff
@@ -0,0 +1,187 @@
+/*
+ * Copyright (c) 2020-2024 Key4hep-Project.
+ *
+ * This file is part of Key4hep.
+ * See https://key4hep.github.io/key4hep-doc/ for further info.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#ifndef LumiCalClusterer_h
+#define LumiCalClusterer_h 1
+
+// clustering options
+#define _CLUSTER_MIDDLE_RANGE_ENGY_HITS 1
+#define _MOLIERE_RADIUS_CORRECTIONS 1
+#define _CLUSTER_MIXING_ENERGY_CORRECTIONS 1
+
+// verbosity
+#define _GENERAL_CLUSTERER_DEBUG 0
+#define _CREATE_CLUSTERS_DEBUG 0
+#define _CLUSTER_BUILD_DEBUG 0
+#define _VIRTUALCLUSTER_BUILD_DEBUG 0
+#define _MOL_RAD_CORRECT_DEBUG 0
```

Comment:
I think these go against identifier rules in c++. Identifiers starting with an underscore followed by an uppercase letter should not be used. Also do these have to be done via pre-processor defines? Could some of them be done via `constexpr`? Or maybe even moved to be run-time configuration?